cmd/edgectl/aes_install.go

```diff
@@ -319,6 +319,50 @@ func (i *Installer) CheckACMEIsDone(hostname string) error {
 	return nil
 }
 
+// GetAESCRDs returns the names of the AES CRDs available in the cluster (and
+// logs them as a side effect)
+func (i *Installer) GetAESCRDs() ([]string, error) {
+	crds, err := i.CaptureKubectl("get AES crds", "", "get", "crds", "-lproduct=aes", "-o", "name")
+	if err != nil {
+		return nil, err
+	}
+	res := make([]string, 0, 15)
```

Comment:
Is there a reason to use 15 cap? It seems brittle and could cause issues for misconfigured/duplicated CRDs and won't play nicely if we ever reach that number of CRDs.

Reply:
Have I misunderstood how that works? I thought I was just suggesting an initial capacity for the slice, but that `append` will increase it (by allocating a new array) as needed.

Reply:
You are right, append will reallocate the array.